src/components/ActiveTask/ActiveTask.jsx

```diff
@@ -27,6 +27,24 @@ export default function ActiveTask ({
   const { t }   = useTranslation('task');
   const classes = useStyles();
 
+  const handleSubmit = useCallback(() => {
+    onNextTaskButtonClick();
+    handleNotification('success', t('notifications:thank_you_for_contributing'));
+  }, [onNextTaskButtonClick, handleNotification, t]);
+
+  // Listen to submit message from iframe: window.parent.postMessage('submit', '*')
+  useEffect(() => {
+    const iframeListener = ({ data }) => {
```

Comment:
Comment could be obsolete if you rename the function to iframeSubmitMessageListener

Reply:
I agree that the comment is obsolete, the code explains the expected message